src/main/java/utils/Wait.java

```diff
@@ -0,0 +1,40 @@
+package utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+public class Wait {
+
+    private WebDriver driver;
+
+    private static final Wait instance= new Wait();
+
+    private Wait(){
+        driver = BrowserManagement.getDriver();
+    }
+
+    public static Wait getInstance() {
+        return instance;
+    }
```

Comment:
I think you wanted to make this a singleton. Check the line 16 of this file, where you do `instance = new Wait():`. To make this a singleton, that line should be here. 
Check another implementations like `getDriver()` and replicate it

Reply:
Fixed getInstance for make this a singleton,